lib/main.js

```diff
@@ -161,6 +161,10 @@ let load = loader({
         }
       }
 
+      if (process.env.FORCE_JS_PG_CLIENT) {
```

Comment:
Why not make this a config.yml thing?

Reply:
it's only useful to get better errors when a SQL query fails to parse, only really useful when doing development.  I'd rather not bake that into the config.yml, since we really should be doing all testing, everwhere with PG-Native.  Doing this is explicitly an attempt to make this environment variable non-discoverable.

I'd rather delete this from the code altogether than put it in config.yml